test/test_runner.go

```diff
@@ -61,6 +63,27 @@ type Result struct {
 	Error    error
 }
 
+// LogCollectionHook can be attached to zerolog.Logger objects, in order
+// to aggregate the log messages in a string slice, containing only the
+// string message.
+type LogCollectionHook struct {
+	Logs []string
+}
+
```

Comment:
Maybe ensures that this conforms to `zerolog.Hook` interface.
```suggestion
var _ zerolog.Hook = &LogCollectionHook{}
```

Reply:
Nice idea :+1: Added in https://github.com/onflow/cadence-tools/pull/114/commits/6dc80945d921db65d74828c695b90a6efe4d9fa5